core/src/main/resources/com/linecorp/armeria/public_suffixes.txt

```diff
@@ -986,7 +986,6 @@ boxfuse.io
 bozen-sudtirol.it
```

Comment:
What's this file? It gets updated when I run the build

Reply:
Me too 😅

Reply:
It's what's retrieved automatically from https://publicsuffix.org/ for handling cookies correctly.